cumulusci/tasks/metadata_etl/base.py

```diff
@@ -20,7 +20,7 @@ class MetadataOperation(enum.Enum):
     RETRIEVE = "retrieve"
 
 
-class BaseMetadataETLTask(BaseSalesforceTask, metaclass=ABCMeta):
+class BaseMetadataETLTask(BaseSalesforceApiTask, metaclass=ABCMeta):
```

Comment:
I'm hesitant about adding this for all Metadata ETL tasks. It makes them set up the REST API clients in `_init_task` but many Metadata ETL tasks only use the Metadata API and don't need `self.sf`/`self.tooling`. Can it be added for the specific task class where you are using the REST or Tooling API instead?

Reply:
This was my (perhaps overambitious) suggestion. I suspected that Beth's problem here was one we'd continue to hit as we build more and more sophisticated MDETL classes, and that a situation where some MDETL tasks but not others could use `self.sf` was a potential source of confusion later.

Do you see a potential harm here or is the risk more in unneeded complexity?

Reply:
@davidmreed I was thinking that there was some scenario where this could make unnecessary requests to Salesforce for authentication, but I checked and we're always initializing simple-salesforce with an instance_url and session_id (access token), which avoids that. So I think it's pretty innocuous and probably worth adding.